CHANGES.md

```diff
@@ -8,6 +8,8 @@
   https://github.com/cnpem/epics-in-docker/pull/57
 * base: add IOCStats module. by @gustavosr8 in
   https://github.com/cnpem/epics-in-docker/pull/62
+* base: add IPMIComm module. by @gustavosr8 in
+  https://github.com/cnpem/epics-in-docker/pull/62
```

Comment:
Nit: I'd rather have a single line in the change list, but that's not mandatory.